datastream/dataset.py

```diff
@@ -156,6 +172,58 @@ def subset(
             functions=self.functions,
         )
 
+    def split(
+        self,
+        key_column: str,
+        proportions: Dict[str, float],
+        filepath: Optional[Union[str, Path]] = None,
```

Comment:
needs an explanation on what happens if the 3 cases
1. not set
2. set but not exist
3. set and exists

Reply:
> Optionally you can chose to stratify on a column in the source dataframe or save the split to a json file.

I think it is explained but I'm biased here.

I think the name `filepath` forces you to look at the documentation though so maybe it should be called `save_filepath` or something?

Reply:
will the split append or overwrite I think is the big thing you need to explain
its clear if you continue reading, but it should be clear directly, it was the first question I had